src/Dfc.CourseDirectory.Services/BulkUploadService/ApprenticeshipBulkUploadService.cs

```diff
@@ -1289,6 +1180,12 @@ private static string Base64Encode(IReaderRow row)
 
             foreach (var record in records)
             {
+                if (record.FRAMEWORK_CODE.HasValue ||
+                    record.FRAMEWORK_PATHWAY_CODE.HasValue ||
+                    record.FRAMEWORK_PROG_TYPE.HasValue)
+                {
+                    continue;
```

Comment:
Perhaps worth a comment in here to say that failing silently for frameworks is intended behaviour sic. PTCD-694?